tests/performance_tests/performance_test.py

```diff
@@ -103,7 +107,7 @@ def performPtbReverseLinearSearch(tpsInitial: int, step: int, testHelperConfig:
 
         myTest = PerformanceBasicTest(testHelperConfig=testHelperConfig, clusterConfig=testClusterConfig, targetTps=searchTarget,
                                     testTrxGenDurationSec=testDurationSec, tpsLimitPerGenerator=tpsLimitPerGenerator,
-                                    numAddlBlocksToPrune=numAddlBlocksToPrune, rootLogDir=testLogDir, delReport=delReport, quiet=quiet, delPerfLogs=delPerfLogs)
+                                    numAddlBlocksToPrune=numAddlBlocksToPrune, rootLogDir=testLogDir, delReport=delReport, quiet=quiet, delPerfLogs=delPerfLogs, oldNodeos=oldNodeos)
```

Comment:
Shouldn't `oldNodeos` be set on the `clusterConfig` object (`testClusterConfig`) instead of being an additional argument to `PerformanceBasicTest`?  That would allow the `Utils.getNodeosVersion().split('.')[0] == "v2"` check to be done once when setting up `testClusterConfig`.